manon/navigation-collapsible-menu.scss

```
@@ -163,9 +163,10 @@ body > header nav {
     &[aria-expanded="false"] {
       & ~ * {
         display: flex;
-        flex-direction: row;
-        justify-content: space-between;
-        align-items: center;
+        flex-direction: var(--navigation-collapsible-menu-uncollapsed-flex-directionw);
+        justify-content: var(--navigation-collapsible-menu-uncollapsed-justify-content);
+        align-items: var(--navigatino-collapsible-menu-uncollapsed-align-items);
```

I think this is an typo:
```suggestion
        align-items: var(--navigation-collapsible-menu-uncollapsed-align-items);
```

---

Ah yes, thanks for catching that. Solved it in my upcoming commit.